# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvi/no-snprintf.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./PORT/clib/snprintf.c.orig	Wed Mar 21 12:42:09 2001
+++ ./PORT/clib/snprintf.c	Wed Mar 21 12:43:28 2001
@@ -1,3 +1,6 @@
+/* This is not needed. - Clifford */
+#if 0
+
 #include <sys/types.h>
 #include <sys/cdefs.h>
 
@@ -52,3 +55,5 @@
 	return (vsprintf(str, fmt, ap));
 #endif
 }
+
+#endif
